### PR TITLE
[SDPA-4357] Deletion and replacement of unsupported actions.

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-engagement.atlassian.net/browse/SDPA-4357
https://digital-engagement.atlassian.net/browse/SDPA-2901

### Issue
1. >Content that is under a content workflow cannot be directly published and unpublished.
2. no `Archive content` `action` available

### Change
1. remove core actions that editors dont want.
```text
    'system.action.node_make_sticky_action',
    'system.action.node_make_unsticky_action',
    'system.action.node_promote_action',
    'system.action.node_save_action',
    'system.action.node_unpromote_action',
    'system.action.node_publish_action',
    'system.action.node_unpublish_action',
    'system.action.pathauto_update_alias_node',
```
2. create 3 action plugins with yml files.
   1. TideCoreArchiveAction
   2. TideCorePublishAction

https://github.com/dpc-sdp/content-vic-gov-au/pull/902

### CODE REVIEW

File: src/Form/NodeActionForm.php [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $ops = [];
+    $batch_size = 10;
+    foreach (array_chunk(array_keys($this->selection), $batch_size) as $ids) {
+      $ops[] = [get_class($this) . '::doAction', [$ids, $this->action]];
+    }
+    $batch = [
+      'title' => t('Processing all contents'),
```
Comment (GROwen) [inline src/Form/NodeActionForm.php:136]: @vincent-gao I'd suggested changing this to
```suggestion
      'title' => t('Processing selected content'),
```
Content is both a singular and plural noun. Here's a Stack Overflow post with some more info. https://english.stackexchange.com/questions/45039/is-there-a-term-for-words-that-have-identical-singular-and-plural-forms

I used to work with the user that posted that question. What a small world.

Comment (vincent-gao) [reply src/Form/NodeActionForm.php:136]: @GROwen Thanks for the review.I will take a look on `unpublish` requirement.

**Comment by vincent-gao** [comment]
> @vincent-gao I'm not sure the unpublish action is required because there's an outstanding question on https://digital-engagement.atlassian.net/browse/SDPA-4357 that references this [comment on SDPA-2901](https://digital-engagement.atlassian.net/browse/SDPA-2901?focusedCommentId=92129)

@GROwen , after discussed with Jess , we keep `Archive`, `Publish` and `Delete` 3 actions (`Delete` action provided by Drupal Core). 
I have deleted `Upublish` action accordingly.
Thanks Guy.

**Comment by vincent-gao** [comment]
> Logic looks good @vincent-gao there are just a couple of linting issues that CI has picked up. Once those builds are green I think it's good to merge.

cool, Thanks Guy. marked it as ready for test.

**Pull request merged**